src/rcheevos/operand.c

```diff
@@ -231,6 +280,13 @@ int rc_parse_operand(rc_operand_t* self, const char** memaddr, uint8_t is_indire
           self->value.num = (unsigned)value;
       }
       break;
+    case '{': /* variable or accumulator */
```

Comment:
I don't think it's necessary to call out the accumulator here, it's just a special variable.